packages/nimble/tests/testthat/test-macros.R

```diff
@@ -531,7 +531,7 @@ test_that('duplicate nested indices from macro expansion error-trapped correctly
         }
         )), regexp = 'Stopped after processing macros and if/then/else statements')
     )
-    expect_true(grepl("dnorm(mu[i], 1)", out[2], fixed=TRUE)) 
+    expect_true(grepl("dnorm(mu[i], 1)", out[3], fixed=TRUE)) 
```

Comment:
Previously the result of the macro after processing was all on one line
```r
{
for (i in 1:3) for (i in 1:10) x[i] ~ dnorm(mu[i], 1)
}
```

Now it is
```r
{
    for (i in 1:3) for (i in 1:10) {
        x[i] ~ dnorm(mu[i], 1)
    }
}
```

Based on the way the macro code is written I think the updated version is more correct (though they should be equivalent?).